### ISSUE TITLE
remove forus logo from dashboards and landing pages

### ISSUE THREAD

**Issue by maxvisser** [teamforus/forus#354]
## Context/goal: 
Maarten expains it here:
https://github.com/teamforus/forus/issues/246#issuecomment-434420437

We should make a strategy on how to approach this with mail communication and other landing pages etc.

**Comment by jamalv** [comment]
Agreed, in my eyes it can be kind of a "white label" thing. If the forus logo will not be there the customers might also be less triggered to want their own brandig on there which saves us work.

So, we don't really need the logo/branding to be there. It can just be neutral.

**Comment by maartenfv** [comment]
I agree with Jamal that the white label might be the best for now in terms of visual styling.

I think making something like a cms for emails send by the system is a lot of work for now. In the future it would be nice for sponsors to be able change the colors and add their logo for some of the emails so people will know it is something official.

**Comment by maartenfv** [comment]
In terms of making front-ends we should maybe have a meeting about this.

In the very beginning (hackathon) we wanted to make a generic platform including front-end.

Just after we aimed at making a generic back-end and connecting it with an API to any front-end. This would make it possible for each municipality to have its own styling and use the system for one or more policies. It also gave us the possibility to make a generic Forus front-end. This Forus front-end would allow us to list all the sponsors etc. in one place.

Now we have come back to making one generic system with one generic front-end and it has become difficult to adjust it to the wishes of the municipality and potentially other sponsors.

The lack of a clear plan has caused some unclearity within the team. We need to decide what we will offer.

**Comment by maartenfv** [comment]
Made a proposal for meeting today at 15.00. Send an invitation request by mail.

**Comment by jamalv** [comment]
On the topic of the emails: I would say they should have styling, but not branding. So probably even remove the colours, and just have them be neutral. The customer should have the feeling they are getting an email from their municipality, not some cool startup ;) This way, when the municipalities see it, they will also not be triggered. In the future we can add branding / customisation options.

**Comment by jamalv** [comment]
**Also regarding emails:** https://github.com/teamforus/forus/issues/261

**Comment by maxvisser** [comment]
We made white label emails and starting to brand landing pages in style of nijmegen. Also we are going to white label dashboards including signupflow.
